Assets/MixedRealityToolkit/_Core/Resources/Shaders/Text3DShader.shader

```diff
@@ -0,0 +1,116 @@
+﻿// Upgrade NOTE: upgraded instancing buffer 'Props' to new syntax.
```

Comment:
Is this shader / inspector needed in favour of using TXText now?

Reply:
Why don't we just use tmpro? It's free and looks a lot better.

Reply:
We should use TMPro, it's the new standard now that it ships with Unity, and our team is getting ready to move over, but we have some tools that don't support it yet. This is a stop-gap until we get our tools updated or we can draw a line here to say TMPro only beyond this point.

Reply:
@SimonDarksideJ We are using this with TextMesh. The Selawik font is currently in the repo and the materials were using the default Unity Font Shader, which is a bad practice so we added this shader in HTK. I brought in the shader to bring MRTK to HTK parody as far as the materials go. We also need to create TMPro Font assets for these fonts to push TMPro as the new standard. If we remove this shader, we should remove the font materials as well and just have the TMPro Font assets.